packages/api-specification/src/new-window-api.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<meta charset=utf-8>
+
+<title>Symphony Desktop Wrapper API Specification</title>
+<link rel='stylesheet' type='text/css' href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css'></link>
+
+<body class='container'>
+
+<header class="page-header">
+  <nav>
+    <ul class="nav nav-pills pull-right">
+      <li><a href="index.html">Index</a></li>
+    </ul>
+  </nav>
+  <h1>Symphony Desktop Wrapper API Demo</h1>
+</header>
+
+<h2>New Window API</h2>
+
+<h3>Try me</h3>
+
+<form class="form-inline">
+  <div class="form-group">
+    <label>URL</label>
+    <input type="text" class="form-control" placeholder="url" id="url">
+  </div>
+  <button type="button" id='new-window-test' class="btn btn-default">Open Window</button>
+</form>
+
+<h3>Overview</h3>
+
+<p>The window API is used to create a new browser window. <a href='https://developer.mozilla.org/en-US/docs/Web/API/Window/open'>HTML5 Window API</a></p>
+
+<h3>Method Signature</h3>
+
+<pre>
+window.open(url, name, features)
+</pre>
+
+<p>Creates a new window.</p>
```

Comment:
Maybe add that if a window already exists with the same `name` then that window will be reused, rather than a new window being created?